internal/errlog/errlog.go

```diff
@@ -0,0 +1,62 @@
+package errlog
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+)
+
+type forwarder struct{}
+
+func (f *forwarder) Write(p []byte) (n int, err error) {
+	Printf("%s", string(p))
+	return len(p), nil
+}
+
+type errlog struct {
+	log         *log.Logger
+	everWritten bool
+	permLogs    strings.Builder
+	prereqLogs  strings.Builder
+}
+
+var singletone errlog
+
+func init() {
+	singletone.log = log.New(os.Stderr, "", log.LstdFlags)
+	log.SetOutput(&forwarder{})
+
+	PermIssueF("Missing permissions errors:\n")
```

Comment:
move to the flush function before the corresponding flush operation or initiate the string builder with these values

Reply:
1. Can't move to the flush (because it needs to be written before all other perm-issue logs).
2. calling PermIssueF() is exactly initializing it -- there's no New() function for strings.Builder -- you can just call WriteString() on it.

Reply:
eventually they are all written to singletone.log, you can write to it before writing the strings.builder

Reply:
let's continue in slack